StortSpelProjekt/Game/src/Gamefiles/Components/MeleeComponent.h

```diff
@@ -39,12 +39,12 @@ namespace component
 		void Update(double dt);
 		void Attack();
 		
-		void setAttackInterval(float interval);
+		void SetAttackInterval(float interval);
```

Comment:
I think this change made something in main.cpp not to compile. In main you still write this with lowerCase S. I had to comment out that line to compile the code